pkg/universe.dagger.io/alpha/uffizzi/test/test.cue

```diff
@@ -0,0 +1,49 @@
+package uffizzi
+
+import (
+	"dagger.io/dagger"
+	"dagger.io/dagger/core"
+
+	"universe.dagger.io/alpha/uffizzi"
+)
+
+dagger.#Plan & {
+
+	client: filesystem: "./data": read: {
+		contents: dagger.#FS
+		include: ["*", "./..."]
+	}
+
+	client: commands: sops: {
+		name: "sops"
+		args: ["-d", "../../../secrets_sops.yaml"]
```

Comment:
To add `UFFIZZI_PASSWORD` to `tests/secrets_sops.yaml`:

1. Ensure that you have sops CLI installed: https://github.com/mozilla/sops
2. Run the following command:
```shell
cd tests
export SOPS_AGE_KEY_FILE="$(git rev-parse --show-toplevel)/tests/age_key.txt"
sops secrets_sops.yaml
# add UFFIZZI_PASSWORD with the relevant value
# save the secrets_sops.yaml file
```
3. Commit & push your changes
4. When all checks go green, this will be ready to merge

---

⚠️ Anyone will be able to read the committed `UFFIZZI_PASSWORD` value by following the steps above ⚠️

Reply:
Thanks for sharing the steps for adding the secrets. Can the steps you have mentioned here go in the documentation ? If so, I will create another PR for the same.

Thank you !

Reply:
Yes, they definitely should be in the documentation, somewhere on the https://docs.dagger.io/1227/contributing page I think.

Reply:
I will create a PR for this then.

Reply:
https://github.com/dagger/dagger/pull/2916